solidity/contracts/ECDSAKeepRewards.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity ^0.5.4;
+
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/SafeERC20.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "@keep-network/keep-core/contracts/Rewards.sol";
+
+contract ECDSAKeepRewards is Rewards {
+    uint256 constant MAX_UINT20 = 1 << 20 - 1;
+    uint256 constant _minimumKeepsPerInterval = 2;
+    IBondedECDSAKeepFactory factory;
+
+    constructor (
+        address _token,
+        address factoryAddress,
+        uint256 _firstIntervalStart,
+        uint256[] memory _intervalWeights
```

Comment:
why is intervalWeights as a constructor param and _minimumKeepsPerInterval a constant?

Reply:
Good point. I think we are going to make all those parameters constants to minimize the risk of making a mistake during deployment.